persistqueue/sqlackqueue.py

```diff
@@ -0,0 +1,206 @@
+# coding=utf-8
+import logging
+import pickle
+import sqlite3
+import time as _time
+import threading
+import warnings
+
+from persistqueue import sqlbase
+from persistqueue.exceptions import Empty
+
+sqlite3.enable_callback_tracebacks(True)
+
+log = logging.getLogger(__name__)
+
+# 10 seconds internal for `wait` of event
+TICK_FOR_WAIT = 10
+
+
+class SQLiteAckQueue(sqlbase.SQLiteBase):
+    """SQLite3 based FIFO queue."""
+
+    _TABLE_NAME = 'queue'
+    _KEY_COLUMN = '_id'  # the name of the key column, used in DB CRUD
+    _ACK_STATUS_INITED = "0"  # init status
```

Comment:
Please create a new class to contain the consts `_ACK_*` or use the `enum.Enum` class